### PR TITLE
New durability section - in development

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Everything - moved the sections now, indexised and reference work being done too in this pass. Need to resolve Eager as a thing.

### CODE REVIEW

File: product_docs/docs/pgd/5/durability/administering.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+When running a PGD cluster with Group Commit, there are some things you need to
+be aware of when administering the system, such as how to safely shut down and 
+restart nodes.
+
+## Planned shutdown and restarts
+
+When using Group Commit with receive confirmations, take care
+with planned shutdown or restart. By default, the apply queue is consumed
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/administering.mdx:13]: I find "consumed" somewhat irritating - how about "processed"?
```suggestion
with planned shutdown or restart. By default, the apply queue is processed
```

File: product_docs/docs/pgd/5/durability/administering.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+restart nodes.
+
+## Planned shutdown and restarts
+
+When using Group Commit with receive confirmations, take care
+with planned shutdown or restart. By default, the apply queue is processed
+prior to shutting down. However, in the `immediate` shutdown mode, the queue
+is discarded at shutdown, leading to the stopped node "forgetting"
+transactions in the queue. A concurrent failure of the origin node can
+lead to loss of data, as if both nodes failed.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/administering.mdx:17]: Should the last sentence be highlighted into a warning?

File: product_docs/docs/pgd/5/durability/administering.mdx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+To ensure the apply queue gets flushed to disk, use either
+`smart` or `fast` shutdown for maintenance tasks. This approach maintains the
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/administering.mdx:20]: What's the difference between these two shutdown modes?

File: product_docs/docs/pgd/5/durability/administering.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/administering.mdx:17]: I don't feel it should. This is discussing shutdowns and restarts and just citing the worst case.

File: product_docs/docs/pgd/5/durability/commit-scopes.mdx [hunk 4]
```diff
@@ -144,51 +144,27 @@ commit_scope_kind:
     * `max_commit_delay` (interval) &mdash; Maximum delay that can be injected to commit to try to keep within the lag limits.
 
 !!! Note
-    `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause as well, making `CAMO` syntax deprecated.
+`CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause as well, making `CAMO` syntax deprecated.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/commit-scopes.mdx:147]:
```suggestion
The `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with an additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause, making `CAMO` syntax deprecated.
```

File: product_docs/docs/pgd/5/durability/comparing.mdx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+## Comparison
+
+Most options for synchronous replication available to
+PGD allow for different levels of synchronization, offering different
+tradeoffs between performance and protection against node or network
+outages.
+
+The following table summarizes what a client can expect from a peer
+node replicated to after receiving a COMMIT confirmation from
+the origin node the transaction was issued to. For commit scopes the Mode
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/comparing.mdx:15]: This sentence is difficult to parse. How about this?
```suggestion
The table summarizes what a client can expect from a peer node that has
replicated a transaction after receiving a COMMIT confirmation from the origin node.
For commit scopes the Mode 
```

File: product_docs/docs/pgd/5/durability/legacy-sync.mdx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+`off` turns off synchronous replication.
+
+Due to PGD applying the transaction before persisting it, the values `on` and
+`remote_apply` are equivalent for logical replication.
+
+### Comparison
+
+The following table summarizes what a client can expect from a peer node
+replicated to after receiving a COMMIT confirmation from the origin node the
+transaction was issued to. The Mode column takes on different meaning depending
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/legacy-sync.mdx:44]: See my suggestion above on how to simplify this sentence.

File: product_docs/docs/pgd/5/durability/limitations.mdx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+Group Commit, Eager and CAMO transactions aren't currently supported in
+combination with the Decoding Worker feature or with transaction streaming.
+Installations using these commit scopes must keep `enable_wal_decoder` and `streaming_mode`
+disabled for the PGD node group.
+
+[Legacy synchronous replication](legacy-sync) uses a mechanism for transaction
+confirmation different from Group Commit. The two aren't compatible, and you
+must not use them together. Therefore, whenever you Group Commit transactions,
+make sure none of the PGD nodes are configured in `synchronous_standby_names`.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/limitations.mdx:23]: Probably a question for @PJMODOS or @mwanner - could we technically prevent this by raising an error if the user enables legacy synchronous replication?

File: product_docs/docs/pgd/5/durability/administering.mdx (hunk 3 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/administering.mdx:20]: :shrug: No idea.

Comment (LenzGr) [reply product_docs/docs/pgd/5/durability/administering.mdx:20]: This refers to the various [ways a PostgreSQL server can be shut down](https://www.postgresql.org/docs/current/server-shutdown.html). So it may make sense to either link to that document or give a hint that we're talking about PostgreSQL's different shutdown methods here (and not some PGD-specific way).

File: product_docs/docs/pgd/5/durability/commit-scopes.mdx [hunk 8]
```diff
@@ -144,51 +144,27 @@ commit_scope_kind:
     * `max_commit_delay` (interval) &mdash; Maximum delay that can be injected to commit to try to keep within the lag limits.
 
 !!! Note
-    `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause as well, making `CAMO` syntax deprecated.
+The `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with an additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause, making `CAMO` syntax deprecated.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/commit-scopes.mdx:147]: I'm unsure if "syntactic sugar" is an appropriate description here. How about "more readable/convenient" or "easier to understand"?

**Comment by LenzGr** [comment]
@djw-m I would like to support you in getting this one through the finish line - do you think it's ready to be promoted from draft state to soliciting reviews from others?

File: product_docs/docs/pgd/5/durability/index.mdx [hunk 9]
```diff
@@ -2,300 +2,69 @@
...
-performance needs using *commit scopes*.
+## Introducing 
 
-In its basic configuration, EDB Postgres Distributed uses asynchronous
-replication. However, commit scopes can change both the default and the 
-per-transaction behavior. It's also possible to configure the legacy Postgres
-synchronous replication using standard `synchronous_standby_names` in the same
-way as the built-in physical or logical replication. However, commit scopes 
-provide much more flexibility and control over the replication behavior.
+* [Overview](overview) introduces these concepts and some of the essential
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/index.mdx:31]:
```suggestion
* [Overview](overview) introduces the concepts and some of the essential 
```

File: product_docs/docs/pgd/5/durability/index.mdx [hunk 10]
```diff
@@ -2,300 +2,69 @@
...
 
-## Comparison
+* [Legacy Synchronous Replication](legacy-sync) shows how traditional Postgres synchronous operations
+can still be accessed under EDB Postgres Distributed.
 
-Most options for synchronous replication available to
-PGD allow for different levels of synchronization, offering different
-tradeoffs between performance and protection against node or network
-outages.
+* [Internal timing of operations](timing) compared Legacy replication with PGD's async and synchronous
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/index.mdx:63]:
```suggestion
* [Internal timing of operations](timing) compares legacy replication with PGD's async and synchronous
```

File: product_docs/docs/pgd/5/durability/overview.mdx [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+    be in terms of replication before allowing commit to proceed.
+
+-   [Synchronous Commit](synchronous-commit): This kind of commit scope allows for a behaviour where the origin node awaits a majority of nodes to confirm and behaves more like a native Postgres synchronous commit.
+
+
+!!! Note Legacy synchronization availability
+For backward compatibility, PGD still supports configuring synchronous
+replication with `synchronous_commit` and `synchronous_standby_names`. See
+[Legacy synchronous replication](legacy-sync) for more on this option,
+but consider using [Group Commit](group-commit) or [Synchronous_Commit](synchronous_commit).
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/overview.mdx:57]:
```suggestion
but consider using [Group Commit](group-commit) or [Synchronous Commit](synchronous_commit).
```

File: product_docs/docs/pgd/5/durability/overview.mdx [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+!!!
+
+## Terms and definitions
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/overview.mdx:60]: Would it make sense to break out this section into a separate chapter, so it's easier to find and reference in the other durability chapters?

File: product_docs/docs/pgd/5/durability/lag-control.mdx [hunk 13]
```diff
@@ -4,46 +4,76 @@ redirects:
...
+   hasn't been applied to one or more peer nodes.
+
+-  RCO - Resource constraint objective acknowledges that there is finite storage
+   available. In PGD, as lag increases the demands on these storage resources
+   also increase.
+
+-  GEO - Group elasticity objective ensures that any node isn't originating new
+   data at a rate that can't be saved to its peer nodes. 
+
+!!!
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/lag-control.mdx:39]: This part renders badly: https://deploy-preview-4677--edb-docs-staging.netlify.app/docs/pgd/latest/durability/lag-control/#background

File: product_docs/docs/pgd/5/durability/lag-control.mdx [hunk 14]
```diff
@@ -4,46 +4,76 @@ redirects:
...
+!!! information
+
+-  RPO - Recovery point objective specifies the maximum-tolerated amount of data
+   that can be lost due to unplanned events, usually expressed as an amount of
+   time. In PGD, RPO determines the acceptable amount of committed data that
+   hasn't been applied to one or more peer nodes.
+
+-  RCO - Resource constraint objective acknowledges that there is finite storage
+   available. In PGD, as lag increases the demands on these storage resources
+   also increase.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/lag-control.mdx:34]: Somewhat less verbose:
```suggestion
   available. In PGD, the demand for storage resources also increases as lag increases.
```

File: product_docs/docs/pgd/5/durability/lag-control.mdx [hunk 15]
```diff
@@ -4,46 +4,76 @@ redirects:
...
+
+-  GEO - Group elasticity objective ensures that any node isn't originating new
+   data at a rate that can't be saved to its peer nodes. 
+
+!!!
+
+To allow an organization to achieve their objectives, PGD offers Lag Control,
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/lag-control.mdx:41]:
```suggestion
To allow organizations to achieve their objectives, PGD offers Lag Control,
```

File: product_docs/docs/pgd/5/durability/lag-control.mdx [hunk 16]
```diff
@@ -4,46 +4,76 @@ redirects:
...
+also known as Replication Lag Control. This feature provides a means to
+precisely regulate the potential imbalance without intruding on applications, by
+transparently introducing a delay to READ WRITE transactions that modify data.
+This delay, the PGD Commit Delay, starts at 0ms.
+
+Using the LAG CONTROL commit scope kind, you can set a maximum time that commits
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/lag-control.mdx:47]:
```suggestion
Using the LAG CONTROL commit scope kind, you can set a maximum time that
```

File: product_docs/docs/pgd/5/durability/commit-scopes.mdx (hunk 8 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/commit-scopes.mdx:176]: Moved the statement anyway to the CAMO section, removed the future promise and changed to "is mostly an alias for"

File: product_docs/docs/pgd/5/durability/index.mdx [hunk 17]
```diff
@@ -2,300 +2,69 @@
...
-node replicated to after receiving a COMMIT confirmation from
-the origin node the transaction was issued to. The Mode column takes
-on different meaning depending on the variant. For PSR and legacy
-synchronous replication with PGD, it refers to the
-`synchronous_commit` setting. For commit scopes, it refers to the
-confirmation requirements of the
-[commit scope configuration](commit-scopes#configuration).
+* [Limitations](limitations) lists the various combinations of durability options
+which are not currently supported or not possible. Do refer to this before deciding
+on a durability strategy.
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/index.mdx:68]: Moved

File: product_docs/docs/pgd/5/durability/overview.mdx (hunk 11 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/overview.mdx:57]: If we call it Synchronous Commit, what's the difference between it and all the other synchronous commit options which are Group Commit. It needs a better name and a better keyword....

File: product_docs/docs/pgd/5/durability/overview.mdx (hunk 12 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/overview.mdx:60]: Created a Durability Terminology section to distiguish it as such. Currently only details about node naming but am open to suggestions for other terminology to add.

File: product_docs/docs/pgd/5/durability/lag-control.mdx (hunk 13 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/lag-control.mdx:39]: Fixed - though we need to track down the broken !!!info admonition.

File: product_docs/docs/pgd/5/durability/overview.mdx (hunk 11 above)
Comment (LenzGr) [reply product_docs/docs/pgd/5/durability/overview.mdx:57]: Please capture this suggestion in Jira so we can take action.

Comment (LenzGr) [reply product_docs/docs/pgd/5/durability/overview.mdx:58]: Tracked in https://enterprisedb.atlassian.net/browse/BDR-4361

**Comment by LenzGr** [comment]
For reviewing a rendered version of these changes, start here: https://deploy-preview-4677--edb-docs-staging.netlify.app/docs/pgd/latest/durability/

File: product_docs/docs/pgd/5/durability/legacy-sync.mdx [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+Consider using [Group Commit](group-commit) or [Synchronous Commit](synchronous_commit) instead.
+!!!
+
+Postgres provides [physical streaming
+replication](https://www.postgresql.org/docs/current/warm-standby.html#STREAMING-REPLICATION)
+(PSR), which is unidirectional but offers a [synchronous
+variant](https://www.postgresql.org/docs/current/warm-standby.html#SYNCHRONOUS-REPLICATION).
+
+For backward compatibility, PGD still supports configuring synchronous
+replication with `synchronous_commit` and `synchronous_standby_names`. Consider using [Group Commit](group-commit) or [Synchronous Commit](synchronous_commit) instead instead.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/legacy-sync.mdx:16]:
```suggestion
replication with `synchronous_commit` and `synchronous_standby_names`. Consider using [Group Commit](group-commit) or [Synchronous Commit](synchronous_commit) instead.
```

File: product_docs/docs/pgd/5/durability/overview.mdx [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+Commit scopes allow four kinds of controlling durability of the transaction:
+
+-   [Group Commit](group-commit): This kind of commit scope controls which and how many nodes
+    have to reach a consensus before the transaction is considered to be committable
+    and at what stage of replication it can be considered committed. This option also
+    allows you to control the visibility ordering of the transaction.
+
+-   [CAMO](camo): This kind of commit scope is a variant of Group Commit in
+which enables the client take on the responsibility of verifying a commit went
+through before retrying.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/overview.mdx:46]: This sentence is difficult to parse. How about this?
```suggestion
-   [CAMO](camo): This kind of commit scope is a variant of Group Commit in which the client takes responsibility for verifying that a transaction has been committed before retrying.
```

File: product_docs/docs/pgd/5/durability/comparing.mdx [hunk 20]
```diff
@@ -0,0 +1,73 @@
+---
+title: Comparing Durability Options
+navTitle: Comparing
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/comparing.mdx:3]: This provides a bit more context in the navigation:
```suggestion
navTitle: Comparing Durability Options
```

File: product_docs/docs/pgd/5/durability/index.mdx [hunk 21]
```diff
@@ -2,300 +2,78 @@
...
+* [Group Commit](group-commit) focuses on the Group Commit option, where you can
+define a transaction as done when a group of nodes agrees it's done.
 
-PGD nodes take different roles during the replication of a transaction.
-These are implicitly assigned per transaction and are unrelated even for
-concurrent transactions.
+* [CAMO](camo) focuses on the Commit At Most Once option, which allows
+applications to take on the responsibility of verifying that a commit has been
+successful before retying. This ensures that their commits only happen at most
+once.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/index.mdx:54]:
```suggestion
* [CAMO](camo) focuses on the Commit At Most Once option, in which applications take responsibility for verifying that a transaction has been committed before retrying.
This ensures that their commits only happen at most once.
```

File: product_docs/docs/pgd/5/durability/limitations.mdx [hunk 22]
```diff
@@ -0,0 +1,109 @@
...
+
+[Commit At Most Once](camo) (CAMO) is a feature that aims to prevent
+applications committing more than once. If you use this feature, take 
+these limitations into account when planning:
+
+- CAMO is designed to query the results of a recently failed COMMIT on the
+origin node. So, in case of disconnection, code the application to immediately
+request the transaction status from the CAMO partner. Have as little delay as
+possible after the failure before requesting the status. Applications must not
+rely on CAMO decisions being stored for longer than 15 minutes.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/durability/limitations.mdx:64]:
```suggestion
origin node. In case of a connection failure, the application must
request the transaction status from the CAMO partner immediately. Ensure to have as little delay as
possible after the failure before requesting the status. Applications must not
rely on CAMO decisions being stored for longer than 15 minutes.
```

File: product_docs/docs/pgd/5/durability/index.mdx (hunk 21 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/index.mdx:54]: Fixed elsewhere

File: product_docs/docs/pgd/5/durability/comparing.mdx [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+
+**Expect**: The peer node has received the changes and applied them to the tuple
+store. The changes have not been persisted to storage.
+
+Confirmation on replication means the peer has received and applied the changes, but 
+has yet to persist them to disk. 
+
+**For**: A better checkpoint than received with less performance.
+
+**Against**: Less robust when a crash occurs.
```
Comment (PJMODOS) [inline product_docs/docs/pgd/5/durability/comparing.mdx:42]: The difference between received and replicated is that replicated data survives PG crash, but not OS crash, received does not survive even PG crash.

Comment (PJMODOS) [reply product_docs/docs/pgd/5/durability/comparing.mdx:42]: I think we need to do better to document that here

File: product_docs/docs/pgd/5/durability/comparing.mdx [hunk 24]
```diff
@@ -0,0 +1,73 @@
...
+Durable confirmation means that the transaction has been written to the peer
+node's storage. This provides protection against loss of data after a crash and
+recovery of the peer node. With Group Commit, this also means the changes are
+visible.
+
+**For**: More robust, able to recover without retransmission in the event of a crash.
+
+**Against**: Doesn't guarantee consistency.
+
+### ON VISIBLE
```
Comment (PJMODOS) [inline product_docs/docs/pgd/5/durability/comparing.mdx:57]: The consistency in visible case is mainly around failover, i.e. the session that commits, loses connections and reconnects is guaranteed to see the changes it did with visible, but not with durable. The other session's consistency is not really something that matters here.

File: product_docs/docs/pgd/5/durability/group-commit.mdx [hunk 25]
```diff
@@ -37,35 +64,39 @@ scope.
...
 part of the consensus. For more on this approach, see [CAMO](camo).
 
-The raft option uses the built-in Raft consensus to decide whether
-commit can happen. Currently, the global Raft is used. For this to work, the majority
-of nodes across the whole cluster must work.
+The `raft` option uses the PGD's built-in Raft consensus to decide whether
+commit can happen. Currently, the global Raft is used. For this to work, the
+majority of nodes across the whole cluster must work. This option means that all
+commits are effectively ordered and every node moves forward in step with those
+commits at the same rate.
```
[the remark this answers is not in the document]
Comment (PJMODOS) [reply product_docs/docs/pgd/5/durability/group-commit.mdx:83]: I even wonder whether this should be documented as backwards compatibility feature and not for new deployments atm.

File: product_docs/docs/pgd/5/durability/legacy-sync.mdx [hunk 26]
```diff
@@ -0,0 +1,92 @@
+---
+title: Legacy synchronous replication using PGD
+navTitle: Legacy synchronous replication
+---
+
+!!! Warning Important
+Consider using [Group Commit](group-commit) or [Synchronous Commit](synchronous_commit) instead.
```
Comment (PJMODOS) [inline product_docs/docs/pgd/5/durability/legacy-sync.mdx:7]: I would reword this to highly recommend commit scope based synchronous commit, it's same thing with more PGD-aware config, the group commit, whatever, that does not need to be pushed here much.

**Review by PJMODOS — CHANGES_REQUESTED** [review]
First pass review, some changes suggested

**Comment by LenzGr** [comment]
Would it make sense to update the [Terminology chapter](https://deploy-preview-4677--edb-docs-staging.netlify.app/docs/pgd/latest/terminology/) to link to the new content as well? Several terminology items now have dedicated chapters that should be linked there.

File: product_docs/docs/pgd/5/durability/group-commit.mdx (hunk 25 above)
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/group-commit.mdx:83]: Should this be replaced by

"The `raft` option is for backwards compatibility only. It uses PGD's built-in Raft consensus for commit decisions. It is slow and only useful with eager conflict resolution."

**Comment by djw-m** [comment]
> Would it make sense to update the [Terminology chapter](https://deploy-preview-4677--edb-docs-staging.netlify.app/docs/pgd/latest/terminology/) to link to the new content as well? Several terminology items now have dedicated chapters that should be linked there.

I'd likely want to do that after this whole section is published and finalized.

File: product_docs/docs/pgd/5/durability/group-commit.mdx (hunk 25 above)
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/group-commit.mdx:83]: Dropped in suggested text for review.

[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/group-commit.mdx:83]: Also adjusted the commit_scope_rules reference section to match.

File: product_docs/docs/pgd/5/durability/comparing.mdx [hunk 27]
```diff
@@ -0,0 +1,73 @@
+---
+title: Comparing Durability Options
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/comparing.mdx:2]: Could you be more specific about issues like this? Or are all the items of concern already covered in the comments that followed?

**Review by Metaluim — DISMISSED** [review]
LGTM as it stands right now, barring that small clarification that we can elaborate on on the "for" and "against" points of the confirmation levels.

File: product_docs/docs/pgd/5/durability/comparing.mdx (hunk 27 above)
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/comparing.mdx:2]: Updated with these adjustments. We can return to this page later.
